AfricanReligions/localisation/afr_general_localizations.csv

```diff
@@ -324,4 +324,20 @@ secret_religious_society_amazigh_reformed_rank_2_female;Adherent;;;;;;;;;;;;;x
 secret_religious_society_amazigh_reformed_rank_3_male;Preacher;;;;;;;;;;;;;x
 secret_religious_society_amazigh_reformed_rank_3_female;Preacher;;;;;;;;;;;;;x
 secret_religious_society_amazigh_reformed_rank_4_male;Herald;;;;;;;;;;;;;x
-secret_religious_society_amazigh_reformed_rank_4_female;Herald;;;;;;;;;;;;;x
\ No newline at end of file
+secret_religious_society_amazigh_reformed_rank_4_female;Herald;;;;;;;;;;;;;x
+follower_of_yan_riga;Follower of Yan Riga;;;;;;;;;;;;;x
```

Comment:
Let's put these localizations up with the other event modifiers (secret_X_community).